Ladybird/AppKit/UI/LadybirdWebView.mm

```diff
@@ -180,6 +180,21 @@ - (void)handleVisibility:(BOOL)is_visible
     m_web_view_bridge->set_system_visibility_state(is_visible);
 }
 
+- (void)findInPage:(String const&)query
```

Comment:
should probably also copy to the find pasteboard (so that cmd-g / cmd-shift-g works in other apps with the same text).

cmd-e ("copy selection to find pasteboard") would eventually be nice too.

Reply:
👀  TIL cmd-g is a thing, will look into it